packages/jazz-tools/src/tools/coValues/coVector.ts

```diff
@@ -0,0 +1,696 @@
+import type { RawBinaryCoStream } from "cojson";
+import { cojsonInternals } from "cojson";
+import {
+  AnonymousJazzAgent,
+  CoValue,
+  CoValueClass,
+  getCoValueOwner,
+  Group,
+  ID,
+  RefsToResolve,
+  Resolved,
+  SubscribeListenerOptions,
+  SubscribeRestArgs,
+  TypeSym,
+} from "../internal.js";
+import {
+  Account,
+  CoValueJazzApi,
+  inspect,
+  loadCoValueWithoutMe,
+  parseCoValueCreateOptions,
+  parseSubscribeRestArgs,
+  subscribeToCoValueWithoutMe,
+  subscribeToExistingCoValue,
+} from "../internal.js";
+
+/**
+ * CoVectors are collaborative storages of vectors (floating point arrays).
+ *
+ * @category CoValues
+ */
+export class CoVector extends Float32Array implements CoValue {
```

Comment:
Are we overriding all `Float32Array` methods? If so, should we be implementing Float32Array as an interface instead so that we don't accidentally leak Float32Array instance methods if we don't override them?
```suggestion
export class CoVector implements Float32Array, CoValue {
```